materials/ggplot.md

```diff
@@ -65,110 +68,114 @@ ggplot(dm_ts_data, aes(x = year, y = count)) +
 * Group on a single graph
 
 ```
-ggplot(ts_data, aes(x = year, y = count, color = species_id)) +
-  geom_line()
+ggplot(acacia, aes(x = CIRC, y = AXIS1, color = ANT)) +
+  geom_point()
 ```
 
 * Facet specification
 
 ```
-ggplot(ts_data, aes(x = year, y = count)) +
-  geom_line() +
-  facet_wrap(~species_id)
+ggplot(acacia, aes(x = CIRC, y = AXIS1)) +
+  geom_point() +
+  facet_wrap(~ANT)
 ```
 
 > Do Tasks 1-4 in [Exercise 3 - Adult vs Newborn Size]({{ site.baseurl }}/exercises/Graphing-adult-vs-newborn-size-R).
 
 ### Layers
 
 * Usage
-	* `ggplot()` sets defaults for layers
-	* Combine layers with `ggplot()` using `+`
-	* Must have at least one layer to plot
-	* Add additional layers, as necessary
-		* Order matters
+    * `ggplot()` sets defaults for layers
+    * Combine layers with `ggplot()` using `+`
+    * Must have at least one layer to plot
+    * Add additional layers, as necessary
+        * Order matters
 
 * Combining different kinds of layers
 
 ```
-ggplot(ts_data, aes(x = year, y = count)) +
-  geom_line() +
+ant_acacia <- filter(acacia, ANT %in% c("CM", "CS", "TP"))
+ggplot(ant_acacia, aes(x = CIRC, y = AXIS1)) +
   geom_point() +
   geom_smooth(method = "lm") +
-  facet_wrap(~species_id)
+  facet_wrap(~ANT)
 ```
 
 * Combining different data sources
+    * Will default to `ggplot()` mappings
 
 ```
-ggplot(dm_ts_data, aes(x = year, y = count)) +
-  geom_line() +
-  geom_line(data = do_ts_data, aes(x = year, y = count), 
-	color = "red")
+ggplot(acacia, aes(x = CIRC, y = AXIS1)) +
+  geom_point() +
+  geom_point(aes(y = AXIS2), color = "red") +
```

Comment:
Let's show a full set of layer arguments here just to help them generalized:

```
geom_point(data = acacia, aes(x = year, y = AXIS2), color = "red")
```

and then maybe add a modification step like:

Since some of the arguments are the same as in the main `ggplot()` call we can use those instead of specifying them

```
geom_point(aes(y = AXIS2), color = "red")`
```